examples/express/connection-sharing/app.js

```diff
@@ -13,6 +12,4 @@ app.get('/', routes.home);
 app.get('/insert', routes.insert);
 app.get('/name', routes.modelName);
 
-app.listen(8000, function() {
-  console.log('listening on http://localhost:8000');
-});
+app.listen(8000, console.log('listening on http://localhost:8000'));
```

Comment:
Should be `app.listen(8000, () => console.log('listening on http://localhost:8000'));`